src/main/java/mate/academy/hibernate/relations/service/impl/MovieServiceImpl.java

```diff
@@ -1,16 +1,29 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.MovieDao;
+import mate.academy.hibernate.relations.dao.impl.MovieDaoImpl;
+import mate.academy.hibernate.relations.exception.DataProcessingException;
 import mate.academy.hibernate.relations.model.Movie;
 import mate.academy.hibernate.relations.service.MovieService;
+import mate.academy.hibernate.relations.util.HibernateUtil;
+import org.hibernate.SessionFactory;
 
 public class MovieServiceImpl implements MovieService {
+    private SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+    private MovieDao movieDao = new MovieDaoImpl(sessionFactory);
```

Comment:
You should not instantiate `MovieDao` directly inside the service. It's better to inject it through a constructor. You've already done this in line 15, so this line is redundant and can be removed.